barretenberg/cpp/src/barretenberg/dsl/acir_proofs/c_bind.hpp

```diff
@@ -84,4 +84,6 @@ WASM_EXPORT void acir_serialize_proof_into_fields(in_ptr acir_composer_ptr,
 
 WASM_EXPORT void acir_serialize_verification_key_into_fields(in_ptr acir_composer_ptr,
                                                              fr::vec_out_buf out_vkey,
-                                                             fr::out_buf out_key_hash);
\ No newline at end of file
+                                                             fr::out_buf out_key_hash);
+
+WASM_EXPORT void acir_prove_ultra_honk(uint8_t const* acir_vec, uint8_t const* witness_vec, uint8_t** out);
```

Comment:
not sure why I don't have to add the other 2 functions in the hpp, but there aren't any errors from missing them